src/core/lib/event_engine/posix_engine/event_poller_posix_default.cc

```diff
@@ -14,31 +14,42 @@
 
 #include <grpc/support/port_platform.h>
 
+#include <memory>
 #include <string>
+#include <utility>  // IWYU pragma: keep
 
 #include "absl/strings/str_split.h"
 #include "absl/strings/string_view.h"
 
 #include "src/core/lib/config/config_vars.h"
+#include "src/core/lib/event_engine/forkable.h"  // IWYU pragma: keep
 #include "src/core/lib/event_engine/posix_engine/ev_epoll1_linux.h"
 #include "src/core/lib/event_engine/posix_engine/ev_poll_posix.h"
 #include "src/core/lib/event_engine/posix_engine/event_poller.h"
+#include "src/core/lib/gprpp/no_destruct.h"  // IWYU pragma: keep
```

Comment:
Is the IWYU pragma necessary? I don't see any other need for it in the codebase with no_destruct.

Reply:
Removed.